data/web/public/index.php

```diff
@@ -1,127 +1,273 @@
-            background-position: bottom;
-            background-repeat: no-repeat;
-            min-height: 600px;
-            text-align: center;
-        }
-        .jumbotron h1 {
-            font-size: 36px;
-            font-family: inherit;
-            font-weight: inherit;
-            text-align: inherit;
-        }
-        .navbar-brand img {
-            width: 70%;
+$host_name = gethostname();
+$host_name_parts = explode("-", $host_name);
+$ready_name = count($host_name_parts) == 4 ? $host_name_parts[1] : $host_name;
```

Comment:
@kerrermanisNL that's my sloppy code from a while back. Unrelated to this PR. We need to refactor the whole 'custom hostname' situation so that the user is prompted for the project name instead of guessing it from the environment of the vagrant dir checkout. for the project name, we should just get that from 'tag' in /etc/hypernode/common.json instead of parsing it from the hostname.